.github/sync-repo-settings.yaml

```diff
@@ -5,14 +5,12 @@ branchProtectionRules:
 - pattern: main
   isAdminEnforced: true
   requiredStatusCheckContexts:
-    - 'PHP 5.5 Unit Test'
-    - 'PHP 5.6 Unit Test'
-    - 'PHP 5.6 Unit Test protobuf-3.12.2,grpc'
-    - 'PHP 7.0 Unit Test'
-    - 'PHP 7.1 Unit Test'
-    - 'PHP 7.2 Unit Test'
-    - 'PHP 7.3 Unit Test'
-    - 'PHP 7.3 Unit Test protobuf-3.12.2,grpc'
+    - 'PHP 7.4 Unit Test protobuf-3.12.2,grpc'
+    - 'PHP 7.4 Unit Test'
```

Comment:
Why do we have duplicate entries of PHP7.4, one has `protobuf-3.12.2,grpc` other doesn't? Why can we not remove the unversioned one!
Same for php8.2.

Reply:
Because we want to run at least some tests against the `protobuf` and `grpc` extensions